Calculate/Calculator.cs

```diff
@@ -0,0 +1,72 @@
+using System.Globalization;
+namespace Calculate;
+
+public class Calculator
+{
+    public IReadOnlyDictionary<char, Func<int, int, double>> MathematicalOperations {
+        get;
+    } = new Dictionary<char, Func<int, int, double>> {
+        { '+', Add },
+        { '-', Subtract },
+        { '*', Multiply },
+        { '/', Divide }
+    };
+
+    public static double Add(int a, int b)
+    {
+        return a + b;
+    }
```

Comment:
```suggestion
    public static double Add(int a, int b) => a + b;
```
You could simplify this into a single-lined lambda. Feel free to write methods on a single line if they are just performing a single operation such as this.